feat(chat): add copy prompt button to task actions

Juice10 opened the pull request:
Closes: https://github.com/Kilo-Org/kilocode/discussions/850

This commit introduces a new "Copy Prompt" button to the action bar for each chat task. Users can now easily copy the initial prompt of a task to their clipboard with a single click.

The button provides visual feedback by temporarily changing its icon to a checkmark upon a successful copy operation. Internationalization support for the new button's tooltip has been added across all supported languages.

## Context

Based on feedback from this discussion: https://github.com/Kilo-Org/kilocode/discussions/850

## Implementation

<!--

Some description of HOW you achieved it. Perhaps give a high level description of the program flow. Did you need to refactor something? What tradeoffs did you take? Are there things in here which you’d particularly like people to pay close attention to?

-->

## Screenshots

| before | after |
| ------ | ----- |
|   <img width="396" alt="image" src="https://github.com/user-attachments/assets/2a42bd5c-e86c-4fad-aa30-d799ddffb8d1" />     |   <img width="411" alt="image" src="https://github.com/user-attachments/assets/ee3ff2c4-da8c-4780-bd7d-dda376c7772d" />    |

## How to Test

Open chat, click copy button

## Get in Touch

<!-- We'd love to have a way to chat with you about your changes if necessary. If you're in the [Kilo Code Discord](https://discord.gg/fxrhCFGhkP), please share your handle here. -->

Juice10 committed:
feat(chat): add copy prompt button to task actions

This commit introduces a new "Copy Prompt" button to the action bar for each chat task. Users can now easily copy the initial prompt of a task to their clipboard with a single click.

The button provides visual feedback by temporarily changing its icon to a checkmark upon a successful copy operation. Internationalization support for the new button's tooltip has been added across all supported languages.

Juice10 committed:
Create stale-rivers-travel.md

kevinvandijk reviewed (changes requested):
Even though I guess it's technically not the history view anymore here and chat view instead, still: can you refactor this PR to use the `copyPrompt` text in history.json? It will save a lot of merge headaches and I prefer having the same copy in one file instead of spreading it to multiple files. If you disagree with linking it to the history.json files then another option would be to put this into kilocode.json files. I just don't want our own specific copy in locale files whenever it's possible to keep it out. It's a huge headache with upstream merges.

Juice10 committed:
refactor(i18n): move copyPrompt translation from chat to history namespace

Remove duplicate copyPrompt translation key from chat.json files across all locales and update TaskActions component to reference the consolidated key in history namespace.

Juice10 committed:
Revert earlier re-order

Juice10 committed:
Revert earlier re-order

Juice10 commented:
@kevinvandijk should be fixe now

Juice10 committed:
Merge branch 'main' into juice10/copy-prompt
